pkgs/development/python-modules/zigpy-zigate/default.nix

```diff
@@ -20,7 +20,7 @@ buildPythonPackage rec {
   src = fetchFromGitHub {
     owner = "zigpy";
     repo = "zigpy-zigate";
-    rev = version;
+    rev = "v${version}";
     sha256 = "EV6DV+BytUcPMtzYVKDnq/Uv2efg3stjL5uVlL62II4=";
```

Comment:
Hash update missing, fixed in #104280.

Reply:
that's my bad, must have been looking at an old "files changed"

Reply:
Pretty sure It was my fault. Anyway, happens.